EKF/ekf_helper.cpp

```diff
@@ -787,6 +787,34 @@ void Ekf::constrainStates()
 	}
 }
 
+float Ekf::compensateBaroForDynamicPressure(const float baro_alt_uncompensated)
+{
+	// calculate static pressure error = Pmeas - Ptruth
+	// model position error sensitivity as a body fixed ellipse with a different scale in the positive and
+	// negative X and Y directions. Used to correct baro data for positional errors
+	const matrix::Dcmf R_to_body(_output_new.quat_nominal.inversed());
+
+	// Calculate airspeed in body frame
+	const Vector3f velocity_earth = _output_new.vel - _vel_imu_rel_body_ned;
+
+	const Vector3f wind_velocity_earth(_state.wind_vel(0), _state.wind_vel(1), 0.0f);
+
+	const Vector3f airspeed_earth = velocity_earth - wind_velocity_earth;
+
+	const Vector3f airspeed_body = R_to_body * airspeed_earth;
```

Comment:
@kamilritz  What do you think about creating a helper function which returns the relative wind in body frame?

Reply:
Also thought about that. If we want to move the compensation to the delayed fusion horizon, there might be a benefit of not adding a helper function.